docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Technical documentation
+
+This repository publishes technical documentation to the https://grafana.com website.
+The [`publish-technical-documentation-next.yml`](../.github/workflows/publish-technical-documentation-next.yml) commits the contents of the [`/docs/sources`](./sources) directory to the `/content/docs/cloud-onboarding` directory in the [`grafana/website`](https://github.com/grafana/website) repository.
+
+Pushes to the `main` branch that add, delete, or modify files in the `/docs/sources` directory trigger the workflow.
```

Comment:
I am making this suggestion because it's not clear whether "Pushes" is a subject or if there is an implicit subject of "It", and this helps me to read it properly.